project/config/ppsni/config/fastly.settings.yml

```diff
@@ -0,0 +1 @@
+site_id: j7ffeyn5
```

Comment:
fastly.settings.yml should not appear in the 'config' directory - if you have config split set up correctly it should appear in the config/production directory